core/include/usModuleSettings.h

```diff
@@ -43,11 +45,15 @@ US_BEGIN_NAMESPACE
  * - \e US_AUTOLOAD_PATHS A ':' (Unix) or ';' (Windows) separated list of paths
  *   from which modules should be auto-loaded.
  *
+ * \deprecated Use FrameworkFactory::newFramework(std::map<std::string, std::string> configuration) to configure the framework.
```

Comment:
@saschazelzer I've created [an issue](https://github.com/jeffdiclemente/CppMicroServices/issues/5) to track the implementation of framework properties. Please review and let me know what you think. Thanks.